### DISCUSSION TITLE
Welcome to tailwind_ui_components Discussions!

### DISCUSSION THREAD

**Discussion by jsvigneshkanna** [jsvigneshkanna/tailwind_ui_components#146] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jsvigneshkanna** [comment]
Any suggestions on website we are building

**Reply by AmanJain18** [reply]
I thought of it, should i fix navbar position, it look wierd while scrolling in small size devices... content can been seen after passing through navbar

![Screenshot_20221106_114904](https://user-images.githubusercontent.com/89217032/200158466-1e47e852-65eb-4332-97f0-35809e72009c.png)

**Reply by jsvigneshkanna** [reply]
Try making the padding top of content below the navbar
